clients/browser/index.html

```diff
@@ -199,8 +199,15 @@
                 this._blockMined()
             }.bind(this));
 
+            $.consensus.on('established', function() {
+                document.getElementById('txSubmitBtn').removeAttribute('disabled');
+                document.getElementById('mnrStartBtn').removeAttribute('disabled');
+            }.bind(this));
+
             $.consensus.on('lost', function() {
-                $.miner.stopWork()
+                $.miner.stopWork();
+                document.getElementById('txSubmitBtn').setAttribute('disabled');
```

Comment:
`setAttribute` actually needs a second parameter according to the standard. In case of boolean values, this should be the empty string `''`.

> Boolean attributes are considered to be true if they're present on the element at all, regardless of their actual value; as a rule, you should specify the empty string ("") in value (some people use the attribute's name; this works but is non-standard).

[Source](https://developer.mozilla.org/en/docs/Web/API/Element/setAttribute)